datasquare_test.go

```diff
@@ -418,6 +418,9 @@ func BenchmarkEDSRoots(b *testing.B) {
 	}
 }
 
+// computeRowProof returns the merkle root, proof, proofIndex, and numLeaves for
```

Comment:
It's not immediately clear as what "proof" and "proof index" mean or refer to. It would be great to explain them further.

Reply:
I just read the implementation in https://github.com/celestiaorg/merkletree/blob/c33506a7aa26be568fbb9714b3a6f66c1b60d2aa/tree.go#L95-L99 and TBH I don't understand it so closing this PR as unresolved.